chapter0-What-is-Java/terminology.md

```diff
@@ -1,19 +1,21 @@
 **Compiler:** The compiler converts source code into a  low-level program.
```

Comment:
I like to use "translate" instead on converts, but that is it.
"low-level" is not necessary super clear. I like to say a compiler translate code from a language used by humans (source code) to a language executable by a computer (assembly, and ultimately bytes).

Reply:
Can a high level language not be compiled into another high level language?  Or at least, something 'higher' than assembly or bytes?